src/azure-cli/azure/cli/command_modules/serviceconnector/_credential_free.py

```diff
@@ -215,7 +215,7 @@ def create_aad_user_in_mysql(self, connection_kwargs, query_list):
                             for d in installed_packages)
         if not pym_installed:
             import pip
-            pip.main(['install', 'mycli'])
+            pip.main(['install', 'pymysql'])
```

Comment:
`mycli` internally depends on `PyMySQL`:

https://github.com/dbcli/mycli/blob/bc9492a2d425c4d4908c1bfc97c316739a284b9f/setup.py#L27

```py
install_requirements = [
    'click >= 7.0',
    # Temporary to suppress paramiko Blowfish warning which breaks CI.
    # Pinning cryptography should not be needed after paramiko 2.11.0.
    'cryptography == 36.0.2',
    # 'Pygments>=1.6,<=2.11.1',
    'Pygments>=1.6',
    'prompt_toolkit>=3.0.6,<4.0.0',
    'PyMySQL >= 0.9.2',
```

I guess that's why `mycli` can be replaced by `PyMySQL` here.

I am still wondering why `mycli` was chosen in lieu of `PyMySQL` initially. Any functionality from `mycli` is used?